upnotify-bot/pom.xml

```diff
@@ -25,7 +37,19 @@
 	    <artifactId>jsoup</artifactId>
 	    <version>1.13.1</version>
 	</dependency>
-	
+
+	  <dependency>
+		  <groupId>org.seleniumhq.selenium</groupId>
+		  <artifactId>selenium-java</artifactId>
+		  <version>4.0.0-alpha-5</version>
+	  </dependency>
+	  <dependency>
+		  <groupId>junit</groupId>
+		  <artifactId>junit</artifactId>
+		  <version>RELEASE</version>
```

Comment:
Please replace 'RELEASE' with a handwritten version, so that a maven update won't possibly harm the functionality of the code